source/Nuke.Common/ProjectModel/SolutionSerializer.cs

```diff
@@ -91,8 +91,15 @@ string TextPattern(string name)
             var projectRegex = new Regex(
                 $@"^Project\(""{GuidPattern("typeId")}""\)\s*=\s*{TextPattern("name")},\s*{TextPattern("path")},\s*""{GuidPattern("projectId")}""$");
 
-            var configurations = content
```

Comment:
```
var configurations = (content.GetGlobalSection("ProjectConfigurationPlatforms") ?? content.GetGlobalSection("ProjectConfiguration")).NotNull()
```

would be enough

Reply:
In that case, don't we want a better error message in case the solution does not contain any  configurations? (Probably not a common scenario, but still)

Reply:
Made the suggested change.